ci/verify-chefdk.bat

```diff
@@ -10,5 +10,12 @@ SET TMP=%TMP%\cheftest
 RMDIR /S /Q %TEMP%
 MKDIR %TEMP%
 
+REM ; Ensure the calling environment (disapproval look Bundler) does not
+REM ; infect our Ruby environment created by the `chef` cli.
+FOR %%E IN (_ORIGINAL_GEM_PATH, BUNDLE_BIN_PATH, BUNDLE_GEMFILE, GEM_HOME, GEM_PATH, GEM_ROOT, RUBYLIB, RUBYOPT, RUBY_ENGINE, RUBY_ROOT, RUBY_VERSION) DO SET %%E=
+
+REM ; Ensure the msys2 build dlls are not on the path
+SET PATH=C:\opscode\chefdk\bin
```

Comment:
We don't want to append or prepend. We're trying to ensure that nothing in `C:\msys2` is in the path. This is to ensure that build binaries (like `libwinpthread-1.dll`) are embedded in the ChefDK and not being found on the path.

I tested this with the path like that and it worked fine. That said, I could also add the default path entries if you think that would be better (`C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\`)